### PR TITLE
Support IsFile for Axios

### PR DESCRIPTION
**Pull request by thijscrombeen**
This PR includes Axios support for File downloads.

### CODE REVIEW

File: src/NSwag.CodeGeneration.TypeScript/Templates/Client.ProcessResponse.HandleStatusCode.liquid [hunk 1]
```diff
@@ -19,6 +23,8 @@ return response.blob().then(blob => { return new {{ operation.ResponseClass }}(s
 return {{ Framework.RxJs.ObservableOfMethod }}({ fileName: fileName, data: {% if Framework.Angular.UseHttpClient %}<any>responseBlob{% else %}<any>response.blob(){% endif %}, status: status, headers: _headers });
 {%         elseif Framework.IsAngularJS -%}
 return this.q.resolve({ fileName: fileName, status: status, data: new Blob([response.data]), headers: _headers });
+{%		   elseif Framework.IsAxios -%}
```
Comment (RicoSuter) [inline src/NSwag.CodeGeneration.TypeScript/Templates/Client.ProcessResponse.HandleStatusCode.liquid]: use spaces instead of tabs + align

**Comment by RicoSuter** [comment]
Thanks for the PR. One small comment

File: src/NSwag.CodeGeneration.TypeScript/Templates/Client.ProcessResponse.HandleStatusCode.liquid (hunk 1 above)
Comment (thijscrombeen) [reply src/NSwag.CodeGeneration.TypeScript/Templates/Client.ProcessResponse.HandleStatusCode.liquid]: That should do it

**Comment by RicoSuter** [comment]
Ready to merge?

**Comment by thijscrombeen** [comment]
Yes. Thanks for the quick response.

**Pull request merged**